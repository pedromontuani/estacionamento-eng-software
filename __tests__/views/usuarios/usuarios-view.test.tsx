import React from 'react';
import {rendererCreate} from '../../utils';

import UsuariosView from '../../../src/views/usuarios/usuarios-view';

describe('Testing users view', () => {
  test('should match snapshot', async () => {
    const tree = rendererCreate(<UsuariosView />);
    expect(tree).toMatchSnapshot();
  });
});
