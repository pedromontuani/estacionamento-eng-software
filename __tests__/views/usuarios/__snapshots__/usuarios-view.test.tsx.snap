// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing users view should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 24,
    }
  }
>
  <View
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#1976d2",
        "borderRadius": 20,
        "elevation": 5,
        "height": 30,
        "justifyContent": "center",
        "opacity": 1,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontSize": 14,
          "fontWeight": "500",
        }
      }
    >
      Adicionar usuário
    </Text>
  </View>
  <RCTScrollView
    ListEmptyComponent={
      <Text>
        Nenhum usuário cadastrado
      </Text>
    }
    contentContainerStyle={
      Object {
        "padding": 24,
      }
    }
    data={
      Array [
        Object {
          "email": "admin@estacionamento.com",
          "nome": "Admin",
          "senha": "123456",
          "tipo": "Administrador",
        },
      ]
    }
    getItem={[Function]}
    getItemCount={[Function]}
    keyExtractor={[Function]}
    onContentSizeChange={[Function]}
    onLayout={[Function]}
    onMomentumScrollBegin={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    style={
      Object {
        "marginHorizontal": -24,
      }
    }
    viewabilityConfigCallbackPairs={Array []}
  >
    <View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "white",
              "borderRadius": 8,
              "elevation": 5,
              "flex": 1,
              "flexDirection": "row",
              "height": 62,
              "marginBottom": 12,
              "padding": 12,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "stretch",
                "flex": 1,
                "justifyContent": "space-around",
              }
            }
          >
            <Text>
              Admin
            </Text>
            <Text>
              Administrador
            </Text>
          </View>
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "opacity": 1,
              }
            }
          >
            <Text
              allowFontScaling={false}
              selectable={false}
              style={
                Array [
                  Object {
                    "color": "#c40a0a",
                    "fontSize": 25,
                  },
                  Object {
                    "marginRight": 10,
                  },
                  Object {
                    "fontFamily": "FontAwesome",
                    "fontStyle": "normal",
                    "fontWeight": "normal",
                  },
                  Object {},
                ]
              }
            >
              
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
